packages/web/src/scss/components/TextArea/_TextArea.scss

```diff
@@ -0,0 +1,82 @@
+@use 'sass:map';
+@use 'theme';
```

Comment:
Sass modules (just like ES modules) should be sorted from the farthest to the closest.